tests/test_sidecar2cdxj.py

```diff
@@ -0,0 +1,50 @@
+import json
+import os
+
+import surt
+from unittest.mock import patch
```

Comment:
I think this belongs in the first import group as it's part of the standard library.